addon/webextension/manifest.json

```diff
@@ -1,12 +1,19 @@
 {
-  "name": "WebExtension-study-BrowserActionButtonChoice",
-  "version": "0.2.0",
+  "name": "WebExtension-whiterose",
+  "version": "1.0.0",
   "manifest_version": 2,
+  "permissions": [
+    "<all_urls>",
```

Comment:
is this really a permission?

Reply:
Without this, I get `Error: Missing host permission for the tab` for both the test-page.html (`file://` resource) and a normal webpage (`http(s)://`). Based on the debugger, the content script loads just fine, but the background script doesn't inject the CSS into the page without it. Looking at [MDN](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/permissions) these are called "Host Permissions"; though it's not explicitly cited on the list of reasons why you would want to use a host permission.